frontend/packages/client/src/components/Community/CommunityEditorProfile.js

```diff
@@ -77,7 +77,7 @@ function CommunityEditorProfile({
         if (imageFile.size > MAX_FILE_SIZE) {
```

Comment:
I think this one needs to be `MAX_AVATAR_FILE_SIZE`  and message `The selected file exceeds the 2MB limit.`